blocks/editable/format-toolbar/link-input.js

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
It is a bit weird that link-input is under `format-toolbar`, considering the link input can be used inline or block level, what about placing it together with `url-input`? The current url-input/index.js could be renamed `button.js` maybe.